Issue/34 education content

mechamancy opened the pull request:
# Proposed Changes

* Makes V1 Education sections
* Edits navbar to reflect new pages, adds new content to all pages except SBIR and Due Diligence pages

Fixes #36 .

mechamancy committed:
test commit (1 page)

ensuring commits work + previewing first page work to see if it displays correctly before moving on to other pages before lunch

mechamancy committed:
preliminary local test push

# Proposed Changes

* Adds new pages to Education sections
* Adds page links to Education sub-navbar

Fixes # .

mechamancy committed:
Updates Navbar

# Proposed Change:

* Fixes navbar because I was silly and forgot to add new page links

Fixes #34.

mechamancy committed:
Wrapped up Education Changes

# Proposed Changes
* Includes all content for V1 Education sections
* Changes reflected in new Education pages and Education sub-navbar

Fixes #34.

JimtheBurger commented on adversarial_apps/src/app/education/cfr-title-15/page.tsx:
```diff
@@ -3,8 +3,8 @@ export default function page(){
             <h1 className="text-6xl font-bold pl-5">CFR Title 15</h1>
-            <h4 className="text-xl font-bold pl-5"><br />What is Title 15</h4>
-            <p className = "pl-5">A CFR stands for Code of Federal Regulations and contains 
+            <h4 className="text-xl font-bold pl-5"><br />What is Title 15 about?</h4>
```
I would look into changing this to "h2". Ideally we don't want to skip header numbers. You can check the Home Page I think to see what this looks like regarding headers.

JimtheBurger commented on adversarial_apps/src/app/education/cfr-title-15/page.tsx:
```diff
@@ -15,7 +15,43 @@ export default function page(){
-                patent processing in foreign countries, etc.</p>
+                patent processing in foreign countries, and other such fields.</p>
+
+            <h4 className="text-xl font-bold pl-5"><br />What is CFR Title 15, Part 791? 
```
Same Here

JimtheBurger commented on adversarial_apps/src/app/education/foci/page.tsx:
```diff
@@ -0,0 +1,22 @@
+    return (
+        <main>
+            <h1 className="text-6xl font-bold pl-5">Foreign Ownership Control and Influence (FOCI)</h1>
+            <h4 className="text-xl font-bold pl-5"><br />What is FOCI? Why should I care?</h4>
```
Same here, just look at all the other "h's" on all of your changed files.

JimtheBurger reviewed (changes requested):
Great Work! Everything seems to be working as expected, just got a few alterations based on a conversation I had with Dami and Conner when we were merging the home pages.

mechamancy committed:
Revisions

made revisions based on james' suggestions; mainly just fixing the header levels

after these changes and a successful linter run, will push to development
